landing-page/components/sections/PartnerSection.tsx

```diff
@@ -6,19 +6,21 @@ import Ukm from '../../icons/partners/ukm'
 import MSHack from '../../icons/partners/MSHack'
 
 const partners = [
-  { name: 'Muensterhack', Icon: MSHack },
-  { name: 'Uniklinik Münster', Icon: Ukm },
-  { name: 'Stadt Warendorf', Icon: StadtWarendorf },
+  { name: 'Muensterhack', icon: MSHack, url: 'https://www.muensterhack.de/' },
+  { name: 'Uniklinik Münster', icon: Ukm, url: 'https://www.ukm.de/' },
+  { name: 'Stadt Warendorf', icon: StadtWarendorf, url: 'https://www.warendorf.de/' },
 ]
 
 const PartnerSection = forwardRef<HTMLDivElement>(function PartnerSection(_, ref) {
   return (
     <TitleSection id="partner" ref={ref} title="Our Partners">
       <div className={tw('flex gap-4 pt-4')}>
       {partners.map((partner) => (
-        <div key={partner.name} className={tw('py-2 px-8 bg-white rounded-lg')}>
-          <partner.Icon className={tw('w-44 h-16')} />
-        </div>
+        <a key={partner.name} href={partner.url} target="_blank" rel="noopener noreferrer">
+          <div className={tw('py-2 px-8 bg-white rounded-lg')}>
+            <partner.icon className={tw('w-44 h-16')} />
```

Comment:
interesting that this works, at least for components not accessed using `.` you have to use uppercase letters as lowercase ones are reserved for actual DOM elements (`div`, `img`, ..), I thought this would apply here as well

Reply:
Okay - interesting. The linter is'nt marking that one.

Reply:
if it builds and renders correctly it seems to work, I just always used an uppercase letter because I thought it doesn't work otherwise
thinking about it, I do like it more with an uppercase letter regardless of it being required or not as it states the intent of that property better